hyperspy/_signals/eds_tem.py

```diff
@@ -258,28 +260,32 @@ def get_calibration_from(self, ref, nb_pix=1):
             mp.Acquisition_instrument.TEM.Detector.EDS.live_time = \
                 mp_ref.Detector.EDS.live_time / nb_pix
 
+
     def quantification(self,
-                       intensities,
-                       kfactors,
-                       composition_units='weight',
+                       intensities='auto',
+                       method='auto',
+                       factors='auto',
+                       composition_units='atomic',
                        navigation_mask=1.0,
                        closing=True,
                        plot_result=False,
                        **kwargs):
         """
-        Quantification of intensities to return elemental composition
-
-        Method: Cliff-Lorimer
+        Quantification using Cliff-Lorimer, the zeta factor method, or
+        ionization cross_sections.
 
         Parameters
         ----------
         intensities: list of signal
             the intensitiy for each X-ray lines.
-        kfactors: list of float
-            The list of kfactor in same order as intensities. Note that
-            intensities provided by hyperspy are sorted by the aplhabetical
-            order of the X-ray lines. eg. kfactors =[0.982, 1.32, 1.60] for
-            ['Al_Ka','Cr_Ka', 'Ni_Ka'].
+        method: 'CL' or 'zeta' or 'cross_section'
+            Set the quantification method: Cliff-Lorimer, zeta factor, or
+            ionization cross sections
+        factors: list of float
+            The list of kfactors, zfactors or cross sections in same order as
+            intensities. Note that intensities provided by Hyperspy are sorted
+            by the aplhabetical order of the X-ray lines.
+            eg. kfactors =[0.982, 1.32, 1.60] for ['Al_Ka','Cr_Ka', 'Ni_Ka'].
```

Comment:
Also a tiny inconsistency in spacing after commas... `['Al_Ka', 'Cr_Ka', 'Ni_Ka']`